docker-id/index.md

```diff
@@ -0,0 +1,45 @@
+---
+description:  Docker ID
+keywords:
+- accounts, docker ID, billing, paid plans, support, Cloud, Hub, Store, Forums, knowledge base, beta access
+title: Docker ID Accounts
+---
+
+Your free Docker ID grants you access to Docker services such as the Docker
+Store, Docker Cloud, Docker Hub repositories, and some beta programs. Your Docker ID becomes repository namespace used by hosted services such as Docker Hub and Docker Cloud. All you need is an email address.
+
+This account also allows you to log in to services such as the Docker Support
+Center, the Docker Forums, and the Docker Success portal.
+
+
+## Register for a Docker ID
+
+Your Docker ID becomes your user namespace for hosted Docker services, and becomes your username on the Docker Forums. A Docker ID also
+
+1. Go to the [Docker Cloud sign up page](https://cloud.docker.com).
+
+2. Enter a username, a valid email address, and a password.
+
+3. Click **Sign Up**.
+
+  Docker sends a verification email to the address you provided.
+
+  > **Note**: You must click this link to activate your account. You cannot log in with your Docker ID until you verify your email address.
+
+4. Click the link in the email to verify your email address.
+
+## Login
+
+Once you register for a Docker ID and verify your email address, you can log in
+to Docker services using either the web console with your Docker ID:
+
+![Login using the web interface](images/login-cloud.png)
+
+Or by using the the `docker login` command. (You can read more about `docker login` [here](../engine/reference/commandline/login/) )
```

Comment:
Incomplete sentence. Maybe something like "To log in using the the Docker CLI,..."